ports/rp2/memmap_mp.ld

```diff
@@ -180,6 +180,10 @@ SECTIONS
         *(.uninitialized_data*)
     } > RAM
 
+    .noinit (NOLOAD) : {
```

Comment:
Can't you just use the `uninitialized_data` section?  It should do the same thing, no?

Reply:
That's what I thought, but in my tests it did not seem to make a difference. I might have to test again lest my assumptions have failed.

Reply:
The generated .map file should be enough to tell you if it works or not.

If it does require a new section, I'd suggest calling it `uninitialized_bss`.

Reply:
And it should probably go after the `bss` section, and be 4-aligned.

Reply:
I think `uninitialized_bss` is a good shout, since it isn't *strictly* data (it's not initialized with any known value) that makes it clearer about intent.